shardingsphere-test/shardingsphere-integration-test/shardingsphere-integration-test-scaling/src/test/java/org/apache/shardingsphere/integration/data/pipeline/cases/mysql/MySQLManualScalingIT.java

```diff
@@ -57,20 +55,35 @@ public static Collection<ScalingParameterized> getParameters() {
             if (Strings.isNullOrEmpty(version)) {
                 continue;
             }
-            result.add(new ScalingParameterized(DATABASE, version, "env/scenario/manual/mysql"));
+            for (String scenario : ScalingScenario.listScenario()) {
+                result.add(new ScalingParameterized(DATABASE_TYPE, version, Joiner.on("/").join("env/scenario/manual/mysql", scenario, ScalingScenario.SCENARIO_SUFFIX)));
+            }
         }
         return result;
     }
     
+    @Before
+    public void setUp() {
+        addResource();
+        initShardingAlgorithm();
+        // TODO wait for algorithm init
+        ThreadUtil.sleep(2000);
```

Comment:
Does it need `20` seconds? Is there any way to wait on dynamic seconds within a maximum seconds?

Reply:
OK, I read wrong value by mistake.

Could we provide easier sleep method, e.g. TimeUnit.sleep

Reply:
Sure